draft-ietf-gnap-core-protocol.md

```diff
@@ -3321,25 +3327,31 @@ The AS validates that the token presented is associated with the management
 URL, that the AS issued the token to the given client instance, and that
 the presented key is appropriate to the token.
 
-If the access token has expired, the AS SHOULD honor the rotation request to
-the token management URL since it is likely that the client instance is attempting to
+Note that in many cases, the access token will have expired for regular use. To facilitate
+token rotation, the AS SHOULD honor the rotation request of the expired access token
+since it is likely that the client instance is attempting to
 refresh the expired token. To support this, the AS MAY apply different lifetimes for
```

Comment:
actually, what we're saying here is that the lifetime for token management might be longer that for its use